src/test/java/winstone/HttpConnectorFactoryTest.java

```diff
@@ -63,6 +85,7 @@ public void writePortInFile() throws Exception {
                 String portInFile = reader.readLine();
                 assertEquals(Integer.toString(port), portInFile);
             }
+            assertFalse("Port value should not be empty in any time", futurePort.get().isEmpty());
```

Comment:
Optional feedback for your consideration: the `try` block above could be simplified to:

```java
assertEquals(Integer.toString(port), Files.readString(portFile, StandardCharsets.UTF_8));
```

If you take my suggestion above to use `Files.readString`, this would be consistent. However, not a blocking comment, since this is pre-existing code you did not write.

Reply:
resolved in [586feef](https://github.com/jenkinsci/winstone/pull/249/commits/586feeff71811269c7bc802b44b0b2fb760a4cfb)